extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -757,14 +758,18 @@ export class AmpSlideScroll extends BaseSlides {
 
     if (slideChanged) {
       const name = 'slideChange';
+      const userInteracted = this.userInteracted();
       const event = createCustomEvent(
         this.win,
         `slidescroll.${name}`,
-        dict({'index': newIndex})
+        dict({'index': newIndex, 'userInteracted': userInteracted})
       );
       this.action_.trigger(this.element, name, event, ActionTrust.HIGH);
 
-      this.element.dispatchCustomEvent(name, {index: newIndex});
+      this.element.dispatchCustomEvent(name, {
+        index: newIndex,
+        userInteracted,
```

Comment:
Were you planning on following `isTrusted` or the "ActionTrust" API?

Reply:
New changes expose as `ampTrust` and follow the high/low/default value pattern.

Reply:
Cool, I think this is a good choice. This might be a good chance to spitball a minor API design discussion. 

Here's the proposed [User Activation v2](https://www.chromestatus.com/feature/5722065667620864) API for comparison: https://mustaqahmed.github.io/user-activation-v2/

Reply:
Something like `HasSeenUserActivation` could be applicable to carousel for _most_ cases, and it certainly does for video components. The more granular state to expose may be `HasConsumableUserActivation` due to the availability of the `toggleAutoplay` action, in which case toggling `autoplay` on could reset the user interaction state for each slide advance.

Reply:
Per conversation, let's add a unit test to make sure we don't accidentally regress this for users of this programmatic API.

Also, should we pass the numeric enum value instead of the string? If we decide to add another `ActionTrust` enum value in the future, what are the implications of either?

Optional: `s/ampTrust/actionTrust`